tests/test_range.py

```diff
@@ -1205,6 +1204,89 @@ def test_fractional_duration(self):
                           datetime(2017, 6, 16, 19, 37, 19, 500000),
                           now)
 
+    def test_workday(self):
+        with freeze_time('2017-06-16 13:40:26'):  # Friday
+            self.assert_range('this workday', datetime(2017, 6, 16), datetime(2017, 6, 17))
+            self.assert_range('previous workday', datetime(2017, 6, 15), datetime(2017, 6, 16))
+            self.assert_range('previous 2 workdays', datetime(2017, 6, 14), datetime(2017, 6, 16))
+            self.assert_range('previous 3 workdays', datetime(2017, 6, 13), datetime(2017, 6, 16))
+            self.assert_range('previous 4 workdays', datetime(2017, 6, 12), datetime(2017, 6, 16))
+            self.assert_range('previous 5 workdays', datetime(2017, 6, 9), datetime(2017, 6, 16))
```

Comment:
This range includes weekend days. This could be an issue if I say, "Give me the events from the previous 5 workdays" but the range includes the weekend so I also get events over the weekend as well.

same for previous 6 workdays as well

Reply:
The `Range` class represents a contiguous time interval. Modeling non-contiguous ranges would be a huge change that would take lots of design and time. Meanwhile, I think it is useful to have the start and end of the interval that includes the given work days.

Reply:
we will make another ticket for non-contiguous time range